code/modules/mob/living/simple_animal/bot/honkbot.dm

```diff
@@ -0,0 +1,350 @@
+/mob/living/simple_animal/bot/honkbot
+	name = "\improper honkbot"
+	desc = "A little robot. It looks happy with its bike horn."
+	icon = 'icons/obj/aibots.dmi'
+	icon_state = "honkbot"
+	density = FALSE
+	anchored = FALSE
+	health = 25
+	maxHealth = 25
+	damage_coeff = list(BRUTE = 0.5, BURN = 0.7, TOX = 0, CLONE = 0, STAMINA = 0, OXY = 0)
+	pass_flags = PASSMOB
+	radio_channel = "Service" //Service
+	bot_type = HONK_BOT
+	bot_filter = RADIO_HONKBOT
+	model = "Honkbot"
+	bot_core_type = /obj/machinery/bot_core/honkbot
+	window_id = "autohonk"
+	window_name = "Honkomatic Bike Horn Unit v1.0.7"
+	data_hud_type = DATA_HUD_SECURITY_BASIC // show jobs
+	path_image_color = "#FF69B4"
+
+	var/honksound = 'sound/items/bikehorn.ogg' //customizable sound
+	var/spam_flag = FALSE
+	var/cooldowntime = 30
+	var/cooldowntimehorn = 10
+	var/mob/living/carbon/target
+	var/oldtarget_name
+	var/target_lastloc = FALSE	//Loc of target when arrested.
+	var/last_found = FALSE	//There's a delay
+	var/threatlevel = FALSE
+	var/declare_arrests = FALSE // speak, you shall not, unless to Honk
```

Comment:
Why is there a bunch of vars from security bots here if they're not used by the honkbot?